docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -260,7 +262,11 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
       rtl: showRtl,
     }
 
-    return <Provider theme={newTheme}>{element}</Provider>
+    return (
+      <Provider key={`${examplePath}${showRtl ? '-rtl' : ''}`} theme={newTheme}>
```

Comment:
Perhaps this should be done in the Provider itself?  It seems adding it here would means RTL will work in our doc examples, but not necessarily anywhere else the Provider is used.

Reply:
not sure, as there is no straightforward way to **generate** the key (as we won't like to generate a new one on each rerendering). Should we request client to provide it?